fineftp-server/src/filesystem.cpp

```diff
@@ -164,7 +164,7 @@ namespace Filesystem
     // https://files.stairways.com/other/ftp-list-specs-info.txt
 
     auto now = std::chrono::system_clock::now();
-    time_t now_time_t = std::chrono::system_clock::to_time_t(now);
+    time_t const now_time_t = std::chrono::system_clock::to_time_t(now);
```

Comment:
`time_t const` vs `const time_t` (right vs left const), usually I like the other one better